src/test/locale/sw.js

```diff
@@ -319,32 +319,32 @@ test('calendar day', function (assert) {
     var a = moment().hours(12).minutes(0).seconds(0);
     assert.equal(
         moment(a).calendar(),
-        'leo saa 12:00',
+        'leo saa 12:00 PM',
         'today at the same time'
     );
     assert.equal(
         moment(a).add({ m: 25 }).calendar(),
-        'leo saa 12:25',
+        'leo saa 12:25 PM',
         'Now plus 25 min'
     );
     assert.equal(
         moment(a).add({ h: 1 }).calendar(),
-        'leo saa 13:00',
+        'leo saa 01:00 PM',
```

Comment:
Do you expect `01:00 PM` or `1:00 PM` in Swahili?

Reply:
Both of them are being frequently used, the most popular one is 01:00 PM